datavault/backend.py

```diff
@@ -99,15 +320,19 @@ def _saveData(self, data):
         f.flush()
 
     def addData(self, data):
+        if transpose:
+            raise RuntimeError("Transpose specified for simple data format: not supported")
         if not len(data) or not isinstance(data[0], list):
             data = [data]
         if len(data[0]) != self.cols:
```

Comment:
this gets set in the `IniData` base class, either when the data is loaded for an existing dataset (https://github.com/martinisgroup/servers/blob/u/maffoo/hdf5_data/datavault/backend.py#L132) or in the `initialize_info` method for new datasets (https://github.com/martinisgroup/servers/blob/u/maffoo/hdf5_data/datavault/backend.py#L210)